services/src/main/java/org/keycloak/authentication/authenticators/x509/AbstractX509ClientCertificateAuthenticatorFactory.java

```diff
@@ -87,6 +87,11 @@
             USERNAME_EMAIL_MAPPER
     };
 
+    private static final String[] certificatePolicyModes = {
```

Comment:
Could you please use upper-case in constant names as you did for others?

Reply:
I'm sorry, I don't understand your suggestion: the constant names are already uppercase (CERTIFICATE_POLICY_MODE_ALL, CERTIFICATE_POLICY_MODE_ANY). Are you talking about the constant values ('Any', 'All')? If that's the case then, sure, no problem!

Reply:
I think Pedro meant to use the uppercase for the constant name itself? Probably something like:
```
private static final String[] CERTIFICATE_POLICY_MODES = {
```

Reply:
Oh, sure! I didn't think about it because it was consistent with constant definitions at the lines above, but I will change it. Thanks!